### PR TITLE
Create 2021-01-04-larb-beans.md

### PR DESCRIPTION
**Pull request by zmagg**

### CODE REVIEW

**Comment by zmagg** [comment]
@mookerji  / poke

**Comment by mookerji** [comment]
@zmagg ... poke?

image

File: _posts/salads/2021-01-04-larb-beans.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Ingredients:
+* Cooked beans, a few cups cooked. Hearty beans with a thicker skin work best. Scarlet runner beans are great. Cranberry beans might work but are a little soft. 
+* 1-2 shallots, thinly sliced
+* 1-2 cloves of garlic, minced
+* Fish sauce
+* Rice, 1-2 cups
+* Oil
+* Dried pepper flakes (ancho/ghost/whatever)
+* 2-4 limes/lemons (limes preferrable)
```
Comment (mookerji) [inline _posts/salads/2021-01-04-larb-beans.md]: s/preferrable/preferable

We should also probably *commit* to limes here.

File: _posts/salads/2021-01-04-larb-beans.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+Inspired by Kin Khao's Nam Tok beans. Never got the texture/seasoning to taste quite as crispy, so let's just call these larb beans shall we? Very tasty, very savory, we've been making them regularly and they never last long.
+
+Ingredients:
+* Cooked beans, a few cups cooked. Hearty beans with a thicker skin work best. Scarlet runner beans are great. Cranberry beans might work but are a little soft. 
+* 1-2 shallots, thinly sliced
+* 1-2 cloves of garlic, minced
+* Fish sauce
+* Rice, 1-2 cups
```
Comment (mookerji) [inline _posts/salads/2021-01-04-larb-beans.md]: Any rice? Jasmine rice?

**Pull request merged**